src/App.jsx

```diff
@@ -1,10 +1,64 @@
 import './App.css';
+import axios from "axios";
+import {useState} from "react";
 
 function App() {
 
+    const [countries, setCountries] = useState([]);
+
+    async function fetchCountries() {
+        try {
+            // cca3 (landcode) ook meegenomen om als key te gebruiken
+            const result = await axios.get(`https://restcountries.com/v3.1/all?fields=name,flag,population,region,cca3`);
```

Comment:
Wees ervan bewust dat je met de sort methode de originele array aanpast. Vaak is het handig om een kloon te maken van de originele array om vervolgens op de gekloonde array de methodes op los te gooien. Zo blijft de originele array intact.

hoe kloon je een array? zo dus:

`const clonedArray = [...originalArray]`